master/reference/calicoctl/setup/etcdv2.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Configuring calicoctl - etcdv2 datastore 
+layout: docwithnav
+---
+
+This document covers the configuration options for calicoctl when using an etcdv2 datastore.
+
+## YAML input file
+
+By default `calicoctl` looks for a configuration file at `/etc/calico/calicoctl.cfg`.
+The file location may be overridden using the `--config` option on commands that required
+datastore access.
+
+The config file is a yaml map, with the following supported fields: 
+
+| Field                  | Description    | Examples
+|------------------------|----------------|----------
+| etcdEndpoints   | A comma separated list of etcd endpoints.  | http://etcd1:2379 
+| etcdUsername    | Username for RBAC (optional)               | "user" 
+| etcdPassword    | Password for the given username (optional) | "password"
+| etcdKeyFile     | Path to the etcd key file (optional)       | /etc/calico/key.pem
+| etcdCertFile    | Path to the etcd client cert (optional)    | /etc/calico/cert.pem
+| etcdCACertFile  | Path to the etcd CA file (optional)        | /etc/calico/ca.pem
+
+#### Example file
+
+```yaml
+etcdEndpoints: http://etcd1:2379,http://etcd2:2379,http://etcd3:2379 
+etcdKeyFile: /etc/calico/key.pem 
+etcdCertFile: /etc/calico/cert.pem 
+etcdCACertFile: /etc/calico/ca.pem 
+```
+
+## Environment variables
+
+Configuration may also be specified using the following environment variables.
```

Comment:
Is there an easier way to say this without redundancy? I think the Felix guides used to say something like "Any of the configuration parameters can be passed as Environment variables by capitalizing and adding an underscore before camel cased letters."